main.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"Fibbonaci/Data"
```

Comment:
I'm not sure why you use `Data` package? Just `Fibbonaci` should be enough.

BTW should be `Fibonacci` (two `c`).